### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by yusufnviiri**
-Create contactForm branch.
-Add contact form in index.html
-Emmanuel add style to contact form via pair programming

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -223,12 +228,44 @@ <h2>About </h2><h2> Myself</h2>
...
               <li>
                 <div id="profiler2"><a href="index.html">Skills</a></div>
-               <a href=""><img id="skill" src="images/arrow2.PNG" alt="frameworks" /></a> 
+                <a href=""
+                  ><img id="skill" src="images/arrow2.PNG" alt="frameworks"
+                /></a>
               </li>
             </ul>
           </section>
+          <form action="post" method="post" class="contact-form">
```
Comment (ReshyResh) [inline index.html:237]: - [ ] Please make sure that, as per requirement, you're using the Formspree service to handle the user input.

File: index.html [hunk 2]
```diff
@@ -223,12 +228,44 @@ <h2>About </h2><h2> Myself</h2>
...
+              required
+            /><br />
+
+            <textarea
+              class="textarea"
+              cols="30"
+              rows="10"
+              placeholder="write your message here"
+              maxlength="500"
+            ></textarea
```
Comment (ReshyResh) [inline index.html:260]: - [ ] Please make sure to stick to the template and use the same margins-paddings and button interactions.


| Your design  | Template design |
| ------------- | ------------- |
|   |   |

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @yusufnviiri , 

### Good job so far!
  - ✔ Correct use of Flexbox to lay the elements.
  - ✔ User input has HTML validation.
  - ✔ No linter errors.


#### There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! 👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @yusufnviiri, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**